build

```diff
@@ -20,9 +20,12 @@ for d in networking/plugins/*; do
 	go install ${REPO_PATH}/$d
 done
 
+echo "Building rkt (stage0)..."
+go build -o $GOBIN/rkt ${REPO_PATH}/rkt
+
 if [[ "$OSTYPE" == "linux-gnu" ]]; then
 	echo "Building init (stage1)..."
-	go build -o $GOBIN/init ${REPO_PATH}/stage1/init
+	go install ${REPO_PATH}/stage1/init
```

Comment:
Stage1 needs to be rebuilt whenever the net-plugins or init are modified.  @eyakubovich pointed out to me that `go install` does _the right thing_ whereas `go build` was always producing new binaries.